.changes/v2.20.0/559-features.md

```diff
@@ -0,0 +1 @@
+* Added methods `SetReadOnlyAccessControl` and `IsSharedReadOnly` `Catalog` and `AdminCatalog`, to handle read-only catalog sharing [GH-559]
```

Comment:
```suggestion
* Added methods `SetReadOnlyAccessControl` and `IsSharedReadOnly` for `Catalog` and `AdminCatalog`, to handle read-only catalog sharing [GH-559]
```

Reply:
Fixed. Incidentally, I moved the changelog entries from `v2.20.0` (when they were created!) to the current `v2.22.0`